scripts/backend_tests_incomplete_coverage.txt

```diff
@@ -103,4 +103,5 @@ scripts.linters.pylint_extensions_test
 scripts.linters.pre_commit_linter_test
 scripts.pre_push_hook_test
 scripts.run_e2e_tests_test
+scripts.run_acceptance_tests_test
```

Comment:
These tests should not be in the exclusion list. This PR should add full backend tests for the scripts it creates